pages/common/yank.md

```diff
@@ -0,0 +1,23 @@
+# yank
```

Comment:
Do you know if Yank works on Windows? I can't see any install instructions in the README.

Reply:
AFAIK it doesn't...

Reply:
Then you should duplicate the page separately in linux and osx folders.

Reply:
Makes sense ^^
Does it need to create another pull request?

Reply:
You can just update this pull request itself.